### PR TITLE
LPS-172918 | Master

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPS-194104](https://liferay.atlassian.net/browse/LPS-194104)

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -3480,6 +3480,63 @@ definition {
 		}
 	}
 
+	@description = "LPS-172918 - Verify that can trigger standalone action for a site scoped object"
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:3483]: Verify the user can trigger a standalone action for a site scoped object

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 2]
```diff
@@ -3480,6 +3480,63 @@ definition {
...
+				inputValue = "New Entry Test",
+				objectLabel = "Custom Object 172918",
+				thenAction = "Add an Object Entry",
+				whenAction = "Standalone");
+		}
+
+		task ("When the standalone action is triggered manually") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject172918");
+
+			ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry");
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:3524]: I'm surprised that this works when we don't actually set the panel category key. Nothing to fix here, just an observation

**Review by timpak — CHANGES_REQUESTED** [review]
@victorhcunha please see my comment below

**Review by timpak — APPROVED** [review]
LGTM!

@victorhcunha in the future, if I only request a minor fix from you, you can forward it yourself after completing the revision. It doesn't need to go through me again.

**Comment by timpak** [comment]
ci:forward

**Pull request closed without merging**